package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "truex-shared",
-  "version": "1.0.77",
+  "version": "1.0.79",
```

Comment:
Typo?

Reply:
actually on purpose. @michaelwct has already taken the 1.0.78 tag for his ongoing changes to this repo